src/meta/inode.rs

```diff
@@ -0,0 +1,66 @@
+use crate::color::{ColoredString, Colors, Elem};
+use std::fs::Metadata;
+
+#[derive(Debug, PartialEq, Eq, Copy, Clone)]
+pub struct INode {
+    index: u64,
+    valide: bool,
+}
+
+impl<'a> From<&'a Metadata> for INode {
+    #[cfg(unix)]
+    fn from(meta: &Metadata) -> Self {
+        use std::os::unix::fs::MetadataExt;
+
+        let index = meta.ino();
+
+        Self {
+            index,
+            valide: true,
+        }
+    }
+
+    #[cfg(windows)]
+    fn from(_: &Metadata) -> Self {
+        Self {
+            index: 0,
+            valide: false,
+        }
+    }
+}
+
+impl INode {
+    pub fn render(&self, colors: &Colors) -> ColoredString {
+        if !self.valide {
+            return colors.colorize(String::from("-"), &Elem::SymLink);
```

Comment:
We could probably use `NonFile` for color. A faded color will probably look better.

Reply:
I try added two colors for the inode, as lsd is trying to provide a colorful ls, `NonFile` looks a little bit boring for `inode,name`.

Maybe @Peltoche could help assign a better color for this?

Currenttly impl:
colorful for unix:
![image](https://user-images.githubusercontent.com/3764335/70858889-8141d080-1f45-11ea-8ea6-e9cec974881c.png)

faded for windows:
![image](https://user-images.githubusercontent.com/3764335/70858910-b4845f80-1f45-11ea-9fc9-81efbc5e2769.png)

Reply:
My bad, I meant the faded color only for windows.